pages/register.vue

```diff
@@ -2,60 +2,67 @@
 definePageMeta({
   layout: 'auth'
 })
+
+const schema = createSchema({
+  email: Yup.string().required().email(),
+  name: Yup.string().required(),
+  surname: Yup.string().required(),
+  password: Yup.string().min(10).required(),
```

Comment:
te same zasady co przy zmianie hasla musisz dac. Mozesz to wyeksportowac np z jakiegos composabla

Reply:
ooo nie przeczytałem tego a tak zrobiłem